### PR TITLE
Create exception for natural language test

### PR DESCRIPTION
**Pull request by BrianEllwood**
the super linter natural language check has been alerting on the word README in the README.me for this repo

This PR creates a Textlint exception to skip the occurrences of this word.

### CODE REVIEW

**Review by Gary-H9 — APPROVED** [review]
LGTM

File: .github/workflows/super-linter.yml [hunk 1]
```diff
@@ -32,4 +32,6 @@ jobs:
         uses: super-linter/super-linter/slim@b4515bd4ad9d0aa4681960e053916ab991bdbe96 # v6.8.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          LINTER_RULES_PATH: /
```
Comment (jacobwoffenden) [inline .github/workflows/super-linter.yml:35]:
```suggestion
```
This is not required if you move `.textlintrc` to `.github/linters`

File: .github/workflows/super-linter.yml [hunk 2]
```diff
@@ -32,4 +32,6 @@ jobs:
...
+          NATURAL_LANGUAGE_CONFIG_FILE: .textlintrc
```
Comment (jacobwoffenden) [inline .github/workflows/super-linter.yml:36]:
```suggestion
```
This is also not needed because it is the default place it looks as per the docs

**Review by jacobwoffenden — CHANGES_REQUESTED** [review]

File: .github/workflows/super-linter.yml (hunk 1 above)
Comment (BrianEllwood) [reply .github/workflows/super-linter.yml]: moved and removed

File: .github/workflows/super-linter.yml (hunk 2 above)
Comment (BrianEllwood) [reply .github/workflows/super-linter.yml]: removed

**Review by jacobwoffenden — APPROVED** [review]

**Pull request closed without merging**